syncopy/datatype/methods/selectdata.py

```diff
@@ -363,6 +365,23 @@ def selectdata(data,
     selectMethod.compute(data, out, parallel=kwargs.get("parallel"),
                          log_dict=log_dct)
 
+    # Handle selection of waveform for SpikeData objects
+    if type(data) == spy.SpikeData and data.waveform is not None:
+        if inplace:
+            spy.log("Inplace selection of SpikeData with waveform not supported for the waveform.", level="INFO")
```

Comment:
So the only real use case for inplace selections from the user side is for frontend runs. I could imagine some `process_metadata` hook which still takes care of the waveform after the computations then? But I am just wildly guessing here, I don't think we would absolutely need this, but maybe escalate to `Warning` level in the log?!

Reply:
it should be possible to add such a hook, but I would suggest not to worry about that unless we really need it (and get complaints from Kathreen or other users). I have increased the log level though, so ppl are aware of what is going on.